src/CommunityToolkit.Maui.MediaElement/Extensions/MetaDataExtensions.macios.cs

```diff
@@ -0,0 +1,221 @@
+﻿
+using AVFoundation;
+using CommunityToolkit.Maui.Core;
+using Foundation;
+using MediaPlayer;
+using UIKit;
+using CoreMedia;
+
+namespace CommunityToolkit.Maui.Extensions;
+
+/// <summary>
+/// The class that provides methods to update the system UI for media transport controls to display media metadata.
+/// </summary>
+public class MetaDataExtensions : IDisposable
+{
+	/// <summary>
+	/// The metadata for the currently playing media.
+	/// </summary>
+	public readonly MPNowPlayingInfo NowPlayingInfo;
+	readonly MPNowPlayingInfo nowPlayingInfoDefault;
+	readonly PlatformMediaElement? player;
+	readonly MPMediaItemArtwork defaultImage;
```

Comment:
let's improve here as well, since the default will not access anything on the instance class, I believe we can make this `static`
```suggestion
	static readonly MPNowPlayingInfo nowPlayingInfoDefault;
	readonly PlatformMediaElement? player;
	static readonly MPMediaItemArtwork defaultImage;
```